packages/components/src/TextField/TextField.tsx

```diff
@@ -27,6 +27,11 @@ export type TextFieldProps = Omit<
    * Максимальная длина ввода
    */
   maxLength?: number;
+  /**
+   * @description Параметр для обрезания пробелов в текст филде при вызове onBlur. По-умолчанию true
```

Comment:
description не пишем, дефолтное значение нужно обозначить соответсвующим тегом

```suggestion
   * Параметр для обрезания пробелов в текст филде при вызове onBlur.
   * @default true
```